src/toPropertyPath.ts

```diff
@@ -0,0 +1,7 @@
+import { decodePointerFragment } from './decodePointerFragment';
+
+export function toPropertyPath(path: string) {
+  return decodePointerFragment(path)
```

Comment:
same as above. `decodePointerFragment` can't be used on a full path. You need to split by `'/'` first.